packages/@okta/vuepress-site/code/rest/index.md

```diff
@@ -21,19 +21,26 @@ You need a free Okta developer edition org to get started. Don't have one? [Crea
   > **Note:** Alternatively, you can [create an OAuth 2.0 access token](/docs/guides/implement-oauth-for-okta/) for use with a number of Okta endpoints. This is a feature that allows you to interact with Okta APIs using scoped OAuth 2.0 access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
 
 2. [Install the Postman app](https://www.getpostman.com/apps).
-3. Launch Postman and select **Import** from the **File** menu.
-4. Click **Import From Link** and then paste this link into the box that appears: `https://developer.okta.com/docs/api/postman/example.oktapreview.com.environment`
-5. Click **Import**.
-6. After the import is complete, verify that the `{yourOktaDomain}` environment is selected in the upper-right corner of the page.
-
-  ![Postman app with collections](/img/postman_example_start.png "Points to the box in the upper-right corner that contains environments for use with Postman")
-
-7. Click the gear icon to the right of `{yourOktaDomain}` and select your org to replace or add these values:
-    * Rename your environment to something that you recognize, for example: `My Org`.
-    * `url`: Replace the example value with your org's full URL: For example, `https://mytestorg.oktapreview.com`. Make sure that you don't include `-admin` in the subdomain.
-    * `apikey`: Enter the API token that you created earlier, for example: `00LzMWxMq_0sdErHy9Jf1sijEGexYZlsdGr9a4QjkS`.
-
-8. Click **Update** to save your changes and then close the **Manage Environments** window.
+3. Start Postman if it's not open already.
+4. At the upper-left corner, click **Import**.
+![Postman window](/img/postman_import_button.png "Displays an arrow pointing to the Import button at the upper-left corner of the Postman window")
+5. In the **Import** dialog box, click **Link**, then paste this link: `https://developer.okta.com/docs/api/postman/example.oktapreview.com.environment` in the **Enter a URL** box and click **Continue**.
```

Comment:
```suggestion
5. In the **Import** dialog box, click **Link** and then paste the following link into the **Enter a URL** box: `https://developer.okta.com/docs/api/postman/example.oktapreview.com.environment`.
6. Click **Continue** and then **Import** to confirm your environment import.
```

That step had a lot going on. Better as two - and with the link introduction at the end so the sentence isn't chopped up. Also combined the next step.

I removed the comma and added "and" because we want to emphasize that the two actions are both independent. If they’re not linked by the right conjunction, it implies that one is independent and the other is supportive. It’s also just good global english standard to set up a signifier. Plus, usually two things in a list are linked by a conjunction and not a comma.